### PR TITLE
Update parity-bridges-common

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

**Comment by fewensa** [comment]
Please add `WeightToFee` to common-primitives
https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/primitives/chain-millau/src/lib.rs#L152-L153
Added by https://github.com/paritytech/parity-bridges-common/pull/1109

**Comment by fewensa** [comment]
https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/primitives/chain-millau/src/lib.rs#L251-L252

https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/primitives/chain-millau/src/lib.rs#L98-L103

Add `WITH_<CHAIN>_MESSAGES_PALLET_NAME`  `PAY_INBOUND_DISPATCH_FEE_WEIGHT`

`WITH_<CHAIN>_MESSAGES_PALLET_NAME` must same with [variable in runtime](https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/bin/rialto/runtime/src/lib.rs#L562), will use this to generate StorageKey https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/relays/lib-substrate-relay/src/messages_source.rs#L211-L218

**Comment by AurevoirXavier** [comment]
> Please add `WeightToFee` to common-primitives
> https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/primitives/chain-millau/src/lib.rs#L152-L153
> Added by [paritytech/parity-bridges-common#1109](https://github.com/paritytech/parity-bridges-common/pull/1109)

These locate in runtime/impls.rs

> https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/primitives/chain-millau/src/lib.rs#L251-L252
> 
> https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/primitives/chain-millau/src/lib.rs#L98-L103
> 
> Add `WITH_<CHAIN>_MESSAGES_PALLET_NAME` `PAY_INBOUND_DISPATCH_FEE_WEIGHT`
> 
> `WITH_<CHAIN>_MESSAGES_PALLET_NAME` must be same with [variable in runtime](https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/bin/rialto/runtime/src/lib.rs#L562), will use this to generate StorageKey https://github.com/paritytech/parity-bridges-common/blob/be6555c51b8e95a4a4215b344f44952f7a06d1fd/relays/lib-substrate-relay/src/messages_source.rs#L211-L218

The first one has already been added.

File: node/runtime/pangoro/src/lib.rs [hunk 1]
```diff
@@ -171,7 +180,7 @@ frame_support::construct_runtime!(
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>} = 16,
 
 		BridgePangolinMessages: pallet_bridge_messages::<Instance1>::{Pallet, Call, Storage, Event<T>} = 17,
-		BridgePangolinDispatch: pallet_bridge_dispatch::<Instance1>::{Pallet, Event<T>} = 18,
+		BridgeDispatch: pallet_bridge_dispatch::<Instance1>::{Pallet, Event<T>} = 18,
```
Comment (AsceticBear) [inline node/runtime/pangoro/src/lib.rs:183]: This change is just a rename issue?  The dispatch pallet has no storage.

Comment (AurevoirXavier) [reply node/runtime/pangoro/src/lib.rs:183]: Yes

**Pull request merged**